File: Multi_Language_CodeExamples/c/structs.c
#include <stdio.h>
#include <string.h>

struct Person {
    char name[50];
    int age;
};

int main() {
    struct Person person = {"Alice", 28};
    printf("Name: %s\n", person.name);
    printf("Age: %d\n", person.age);
    return 0;
}

File: Multi_Language_CodeExamples/rust/structs.rs
struct Person {
    name: String,
    age: u32,
}

impl Person {
    fn greet(&self) {
        println!("Hello, I'm {} and I'm {} years old", self.name, self.age);
    }
}

fn main() {
    let person = Person {
        name: String::from("Alice"),
        age: 28,
    };
    person.greet();
}